docs/tutorials/gpu_executor.md

```diff
@@ -0,0 +1,302 @@
+# Executor on GPU
+
+This tutorial will show you how to use an executor on a GPU, both locally and in a
+Docker container. You will also see how to use GPU with pre-built Hub executors.
+
+Using a GPU allows you to significantly speed up encoding for most deep learning models,
+reducing response latency from 100s to merely 10s of miliseconds. Jina enables you to use
+GPUs like you normally would in a python script, or in a docker container - it does not
```

Comment:
**Automated message**

## Capitalization

Please capitalize first letters of the following (when used to reference project or component):

- Flow
- Executor
- Document
- Jina
- Jina Hub
- Python
- Docker
- Kubernetes

### Note!

This only applies **outside** a code or CLI context. If you're describing Python, YAML, any programming language, or command line input/output, please use the capitalization you would use **in that context**.

##### Right

```python
from jina.flow import Flow
```

##### Wrong

```python
from Jina.Flow import Flow
```

---

Made with [GitHub saved replies](https://github.com/settings/replies)